jest.config.js

```diff
@@ -19,10 +19,5 @@ module.exports = {
   },
   transformIgnorePatterns: ["[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$"],
   setupFilesAfterEnv: ["@testing-library/jest-dom", "./scripts/setup-test.ts"],
-  globals: {
-    "ts-jest": {
-      tsconfig: "tsconfig.json",
-    },
-  },
```

Comment:
This setting is unnecessary because we are using `swc/jest`.
See: https://huafu.github.io/ts-jest/user/config/#introduction